pysb/tools/sensitivity_analysis.py

```diff
@@ -112,72 +109,247 @@ class InitialsSensitivity(object):
       [ 5.0243  5.0243  0.      0.    ]
       [-4.5381 -4.5381  0.      0.    ]]
     >>> sens.create_boxplot_and_heatplot() #doctest: +SKIP
+    >>> values_to_sample = [.9, 1.1]
+    >>> sens = PairwiseSensitivity(\
+            values_to_sample=values_to_sample,\
+            observable=observable,\
+            objective_function=obj_func_cell_cycle,\
+            solver=solver,\
+            sens_type='params'\
+        )
+    >>> print(sens.b_matrix)
```

Comment:
Not sure this huge matrix adds much to the example. I'd recommend removing it from the doctest, or changing it to print something shorter (like the matrix shape).